corehq/apps/accounting/views.py

```diff
@@ -1310,6 +1312,32 @@ def post(self, request, *args, **kwargs):
         return self.get(request, *args, **kwargs)
 
 
+class TriggerAutoRenewalView(BaseTriggerAccountingTestView):
+    urlname = 'accounting_test_auto_renew'
+    page_title = "Trigger Subscription Auto-Renewal"
+
+    @property
+    @memoized
+    def trigger_form(self):
+        if self.request.method == 'POST':
+            return TriggerAutoRenewalForm(self.request.POST)
+        return TriggerAutoRenewalForm()
+
+    def post(self, request, *args, **kwargs):
+        if self.async_response is not None:
+            return self.async_response
+        if self.trigger_form.is_valid():
+            domain = self.trigger_form.cleaned_data['domain']
+            auto_renew_subscriptions(domain)
+            messages.success(
+                request,
+                f'Successfully triggered subscription auto-renewal for "{domain}". '
+                'Any eligible subscriptions for this project will be automatically renewed.',
```

Comment:
It looks like the domain can be empty, which would cause us to auto renew all subscriptions, regardless of domain. I assume that isn't an issue, because I'm guessing this is an internal tool? But the message would look a bit awkward if the domain was `""`. We might want to make that a conditional.

Reply:
I had a tough time finding exactly where it's happening, but the field is required in practice. It has the `required` html attr and does validate that it's not empty:
<img width="774" height="227" alt="image" src="https://github.com/user-attachments/assets/5ec6cda9-1442-4494-9557-5991932a4c94" />